package.json

```diff
@@ -46,6 +50,18 @@
     "topics": {
       "app:state": {
         "description": "Manage your App Builder State storage"
+      },
+      "app:db": {
+        "description": "Manage your App Builder Database storage"
+      },
+      "app:db:collection": {
+        "description": "Manage database collections, also available under 'aio app db col <command>'"
+      },
+      "app:db:document": {
+        "description": "Manage documents in a collection, also available under 'aio app db doc <command>'"
+      },
+      "app:db:index": {
+        "description": "Manage indexes on a collection, also available under 'aio app db ind <command>'"
```

Comment:
I think @pdohogne-magento mentioned this elsewhere, but **idx** might be a better short version for **index**. Doing a quick search, **idx** consistently shows up, while **ind** does not seem very common.